docs/grid-functionalities/Row-based-edit.md

```diff
@@ -0,0 +1,70 @@
+#### index
+- [The action column](#the-action-column)
+- [Multiple Row Selections](#multiple-row-selections)
+- [Change Dynamically Single/Multiple Selections](#changing-dynamically-from-single-to-multiple-selections-and-vice-versa)
+- [Mixing Single & Multiple Row Selections](#mixing-single--multiple-row-selections)
+- [Disable Custom Rows Selections via `selectableOverride`](#disable-custom-rows-selections-via-selectableoverride)
+- [Disable External Button when having Empty Selection](#disable-external-button-when-having-empty-selection)
+- [Change Row Selections](#change-row-selections)
+- Troubleshooting
+  - [Adding a Column dynamically is removing the Row Selection, why is that?](#adding-a-column-dynamically-is-removing-the-row-selection-why-is-that)
+
+### Description
+The Row based editing plugin makes it possible to keep the grid readonly except for rows which the user explicitely toggles into editmode.
```

Comment:
is "editmode" a property? If so, wrap it in backtick or if not then maybe separate it in 2 words?